### PR TITLE
Orientation grouping

### PR DESCRIPTION
**Pull request by OSemenovBoyarka**
This request adds grouping of generated screenshots by orientation feature, which was dropped in 1.0 release. 
One thing I'm not sure about is snapshot helper update required detection. 
I've updated helper and incremented it's version in the end of file, but didn't have a chance to check this

### CODE REVIEW

File: lib/assets/SnapshotHelper.swift [hunk 1]
```diff
@@ -56,4 +57,4 @@ class Snapshot: NSObject {
 
 // Please don't remove the lines below
 // They are used to detect outdated configuration files
-// SnapshotHelperVersion [[1.0]]
```
Comment (KrauseFx) [inline lib/assets/SnapshotHelper.swift]: Ah, good point. We also need to add a migration for this

**Comment by KrauseFx** [comment]
Thanks for the pull request. There are a lot of styling issues. Could you run `rubocop -a` to fix those?

**Comment by SemenovAlexander** [comment]
Yep, sorry for that, I'm not familiar with ruby code style and didn't know about that rubocop tool. Pushed style fixes

**Comment by KrauseFx** [comment]
Looks great, thank you so much for your great contribution @SemenovAlexander :+1:

**Comment by KrauseFx** [comment]
Sorry, I had to revert the changes, as it didn't work, it showed all screenshots duplicated for me. Did you try the changes locally?

**Comment by SemenovAlexander** [comment]
Sure, I use it almost every day for my project. Sorry for that, I'll check once more today. 
Should I fire new pull request after fixing that issue, or?

**Pull request merged**